src/main/java/com/exasol/releasedroid/adapter/communityportal/CommunityPortalReleaseMaker.java

```diff
@@ -0,0 +1,74 @@
+package com.exasol.releasedroid.adapter.communityportal;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+import org.json.JSONObject;
+
+import com.exasol.releasedroid.usecases.exception.ReleaseException;
+import com.exasol.releasedroid.usecases.release.ReleaseMaker;
+import com.exasol.releasedroid.usecases.repository.ReleaseLetter;
+import com.exasol.releasedroid.usecases.repository.Repository;
+
+/**
+ * This class is responsible for releasing on Exasol Community Portal.
+ */
+public class CommunityPortalReleaseMaker implements ReleaseMaker {
+    private static final Logger LOGGER = Logger.getLogger(CommunityPortalReleaseMaker.class.getName());
+    private final CommunityPortalGateway communityPortalGateway;
+
+    public CommunityPortalReleaseMaker(final CommunityPortalGateway communityPortalGateway) {
+        this.communityPortalGateway = communityPortalGateway;
+    }
+
+    @Override
+    public void makeRelease(final Repository repository) throws ReleaseException {
+        LOGGER.fine("Creating a draft of the release announcement on the Exasol Community Portal.");
+        try {
+            final var communityPost = getCommunityPost(repository);
+            this.communityPortalGateway.createDraftPost(communityPost);
```

Comment:
postDraftPost also sounds strange because of two posts :)
I re-named it to sendDraftPost for now and I'm going to test some other thing in the next pull request, so probably this method will be renamed to "submitPostForReview" or something like this